SofaKernel/SofaBase/src/SofaBase/initSofaBase.cpp

```diff
@@ -45,6 +46,7 @@ void initSofaBase()
         initBaseLinearSolver();
         initBaseVisual();
         initBaseUtils();
+        initEigen2Solver();
```

Comment:
normal?

Reply:
yes, Eigen2Solver is in SofaBase (and the init..() was in SofaCommon, I dont know why)